cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletTestUtils.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+package com.linkedin.kafka.cruisecontrol.servlet;
+
+import com.linkedin.kafka.cruisecontrol.config.constants.WebServerConfig;
+
+
+public class KafkaCruiseControlServletTestUtils {
```

Comment:
Nit: It is common practice / convention to make utility classes `final`.

Reply:
Thanks for letting me know